rero/rero-ils-ui
# menu: allow the librarian to switch library

Enables the librarian to change from one libary to another
if he or she works in several libraries.

* Fixes patron view to display multiple libraries.
* Fix populate of select in the routing system.
* closes rero/rero-ils#1510

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Dependencies

* https://github.com/rero/ng-core/pull/306

## How to test?

- Check library switch with a system_librarian (All library of current organisation)
- Check library switch with a librarian with multiple libraries (Show only libraries attached)
- Check if the library switch menu is present if the librarian has only one library.

## Code review check list
